fonts/fonts.css

```diff
@@ -0,0 +1,9 @@
+/** Generated by FG **/
+@font-face {
+	font-family: 'Conv_GothaProBla';
+	src: url('fonts/GothaProBla.eot');
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5985384)